whois-commons/src/main/java/net/ripe/db/whois/common/DatabaseDummifierJmx.java

```diff
@@ -72,13 +84,14 @@ public DatabaseDummifierJmx() {
     @ManagedOperationParameters({
             @ManagedOperationParameter(name = "jdbcUrl", description = "jdbc url"),
             @ManagedOperationParameter(name = "user", description = "jdbc username"),
-            @ManagedOperationParameter(name = "pass", description = "jdbc password")
```

Comment:
Can you change the comment too to include the environment? /**
 * in jmxterm, run with:
 *      run dummify jdbc:mariadb://<host>/<db> <user> <pass>
 *
 * in console, run with
 *      java -Xmx1G -cp whois.jar net.ripe.db.whois.common.DatabaseDummifierJmx --jdbc-url jdbc:mariadb://localhost/BLAH --user XXX --pass XXX
 *
 */

Reply:
Done